arcgis-ios-sdk-samples/Analysis/Analyze hotspots/AnalyzeHotspotsViewController.swift

```diff
@@ -68,17 +68,17 @@ class AnalyzeHotspotsViewController: UIViewController, HotspotSettingsVCDelegate
         params.inputs["Query"] = AGSGeoprocessingString(value: queryString)
         
         //job
-        self.geoprocessingJob = self.geoprocessingTask.geoprocessingJobWithParameters(params)
+        self.geoprocessingJob = self.geoprocessingTask.geoprocessingJob(with: params)
         
         //start job
-        self.geoprocessingJob.startWithStatusHandler({ (status: AGSJobStatus) in
+        self.geoprocessingJob.start(statusHandler: { (status: AGSJobStatus) in
```

Comment:
that method has two completion blocks `startWithStatusHandler:completion:`. It works only for the last argument.